services/src/unit-test/java/uk/gov/moj/sdt/services/utils/GenericXmlParserTest.java

```diff
@@ -318,29 +325,31 @@ public void testParseIndividualResponseEmptyDetailSuccess() throws Exception {
      * @throws Exception if there is any IO problems
      */
     @Test
-    public void testParseIndividualUpdateRequestSuccess() throws Exception {
-        genericXmlParser.setEnclosingTag("targetAppDetail");
-        final Map<String, String> replacementNamespaces = new HashMap<String, String>();
-        replacementNamespaces.put("http://ws.sdt.moj.gov.uk/2013/sdt/targetApp/IndvResponseSchema",
+    void testParseIndividualUpdateRequestSuccess() throws Exception {
+
+        genericXmlParser.setEnclosingTag(TARGET_APP_DETAIL);
+        final Map<String, String> replacementNamespaces = new HashMap<>();
+        replacementNamespaces.put(URL_INDV_RESPONSE_SCHEMA,
                 "http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema");
         replacementNamespaces.put("http://ws.sdt.moj.gov.uk/2013/sdt/IndividualUpdateRequestSchema",
                 "http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema");

```

Comment:
Can this be replaced with URL_BULK_FEEDBACK_RESPONSE_SCHEMA?

Reply:
replace

Reply:
Confirmed text has been replaced with constant.